src/main/java/ch/wisv/areafiftylan/utils/mail/MailRestController.java

```diff
@@ -64,9 +76,60 @@ public MailRestController(MailService mailService, UserService userService, Team
         return createResponseEntity(HttpStatus.OK, "Mail successfully sent");
     }
 
+    @RequestMapping(value = "/noticket", method = RequestMethod.POST)
+    ResponseEntity<?> sendMailToUserWithoutTicket(@Validated @RequestBody MailDTO mailDTO) {
+        Collection<User> users = ticketService.getAllTickets().stream()
+                .map(t -> t.getOwner())
+                .distinct()
+                .collect(Collectors.toList());
+
+        Collection<User> usersWithoutTicket = userService.getAllUsers().stream()
+                .filter(u -> !users.contains(u))
+                .collect(Collectors.toList());
+
+        mailService.sendTemplateMailToAll(usersWithoutTicket, mailDTO);
+
+        return createResponseEntity(HttpStatus.OK, "Mail successfully sent");
+    }
+
+    @RequestMapping(value = "/ticket", method = RequestMethod.POST)
+    ResponseEntity<?> sendMailToUserWithTicket(@Validated @RequestBody MailDTO mailDTO) {
+        Collection<User> users = ticketService.getAllTickets().stream()
+                .map(t -> t.getOwner())
+                .distinct()
+                .collect(Collectors.toList());
+
+        mailService.sendTemplateMailToAll(users, mailDTO);
+
+        return createResponseEntity(HttpStatus.OK, "Mail successfully sent");
+    }
+
+    @RequestMapping(value = "/pickup", method = RequestMethod.POST)
+    ResponseEntity<?> sendMailToUserWithPickup(@Validated @RequestBody MailDTO mailDTO) {
+        Collection<User> users = ticketService.getAllTicketsWithTransport().stream()
+                .map(Ticket::getOwner)
+                .collect(Collectors.toList());
+
+        mailService.sendTemplateMailToAll(users, mailDTO);
+
+        return createResponseEntity(HttpStatus.OK, "Mail was successfully sent");
+    }
+
+    @RequestMapping(value = "/user/{seatgroup}", method = RequestMethod.POST)
+    ResponseEntity<?> sendMailToSeatGroup(@RequestParam String seatGroup, @Validated @RequestBody MailDTO mailDTO) {
```

Comment:
Why is this useful? Can't really think of a realistic case where we want to mail specific seatgroups